lib/ProductOpener/Ingredients.pm

```diff
@@ -52,54 +52,53 @@ ingredients (is the product vegetarian, vegan, does it contain palm oil etc.)
 package ProductOpener::Ingredients;
 
 use ProductOpener::PerlStandards;
-use Exporter    qw< import >;
+use Exporter qw< import >;
 
-BEGIN
-{
-	use vars       qw(@ISA @EXPORT_OK %EXPORT_TAGS);
+BEGIN {
+	use vars qw(@ISA @EXPORT_OK %EXPORT_TAGS);
 	@EXPORT_OK = qw(
-		&extract_ingredients_from_image
+	  &extract_ingredients_from_image
```

Comment:
Is it normal that we have some indentations that are tabs, and some that are 2 spaces, sometime on the same line like here? That feels weird.

Reply:
hum, this is not cool indeed.

Reply:
Would it be ok according to you to remove all tabs ? And just use space everywhere ?

Reply:
The algorithm of perltidy seems to be:
- add indentation as space
- convert any leading group of 4 spaces to tab
We can disable tabs if we want.

Here the "continuation indentation" (that is indentation for a split line) is 2 (the default).
I will change that to 4 to be consistent (I think it's better).

Thus we can also keep tabs transformation, as we want.

Reply:
from the documentation :
> Using tab characters will almost certainly lead to future portability and maintenance problems, so the default and recommendation is not to use them. For those who prefer tabs, however, there are two different options.

Reply:
We decided to keep with tab of length 4. Continuation line should be 4.